pay-api/src/pay_api/services/payment_service.py

```diff
@@ -140,6 +142,14 @@ def create_invoice(cls, payment_request: Tuple[Dict[str, Any]], authorization: T
 
         return invoice.asdict()
 
+    @staticmethod
+    def _validate_and_throw_error(payment_method: str, payment_account: PaymentAccount):
```

Comment:
is there a better place/where is the correct place to do this check? @sumesh-aot

Reply:
May be add this in the factory or pad_service invoice creation method ?